CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0
```

Comment:
You'll want to rev the pubspec version as well.

Reply:
You mean update `pubspec.yaml` right? It's already updated: https://github.com/dart-lang/native_synchronization/pull/26/files#diff-8b7e9df87668ffa6a04b32e1769a33434999e54ae081c52e5d943c541d4c0d25R3

Reply:
Oops, yup, so it is.